README.md

```diff
@@ -24,6 +24,7 @@ You also need to install
 In order to generate the CSV file:
 
 1. Rename config_example.ini to config.ini.
+	- In the parse_apartments.py file, declare the absolute path of the config.ini file in main().
```

Comment:
Shouldn't have both a config file and have the user directly edit the python code.